### ISSUE TITLE
Entity .../behaviors/.../0/quantity/avg bug

### ISSUE THREAD

**Issue by NoNameUser9** [anegostudios/VintageStory-Issues#1299]
Game Version: 1.15.5
Platform: Windows 10
Modded: Yes

### Description
if I change the value of the amount of the dropped item in the behavior, then apart from zero (only in /server/behaviors/5/drops/0/quantity/avg), the value will always be more by one

### How to reproduce
Entity patch (
[
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/0/quantity/avg",
	"value": 1,
    "file": "entities/land/chicken-hen.json"
  },
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/0/quantity/var",
	"value": 0,
    "file": "entities/land/chicken-hen.json"
  },
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/1/quantity/avg",
	"value": 1,
    "file": "entities/land/chicken-hen.json"
  },
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/1/quantity/var",
	"value": 0,
    "file": "entities/land/chicken-hen.json"
  },
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/2/quantity/avg",
	"value": 1,
    "file": "entities/land/chicken-hen.json"
  },
  {
    "op": "replace",
    "path": "/server/behaviors/5/drops/2/quantity/var",
	"value": 0,
    "file": "entities/land/chicken-hen.json"
  }
]
)

![2021-09-13_17-03-12_LI](https://user-images.githubusercontent.com/82166402/133098749-ff5c03f5-4ff9-48e5-8ce5-40657b40db76.jpg)

**Comment by tyronx** [comment]
not sure i understand, whats the expected behavior here?

**Comment by NoNameUser9** [comment]
in the first slot, as in the rest, I set the value to 1, but it is always 1 higher except zero

**Comment by NoNameUser9** [comment]
When I set 1, I expect to see 1, but it is not possible to set 1 in the first slot as it is set to +1. That is, if I put 2, then I get 3, if 3, then 4, etc.

**Comment by NoNameUser9** [comment]
This is valid only for the first slot, everything is fine with the rest

**Comment by tyronx** [comment]
oh, there is always one more chicken meat?

**Comment by NoNameUser9** [comment]
Yes exactly. I checked on other entities as well, so this is a problem of the entire system as a whole

**Comment by Craluminum2413** [comment]
Related to #615
